glossary.qmd

```diff
@@ -0,0 +1,353 @@
+
+#### OGR
+
+A widely-used open-source library for converting between different [vector data](#vector-data) formats, as well as reprojecting between [coordinate reference systems](#coordinate-reference-system-crs).
+
+Its common command-line tools include `ogrinfo` and `ogr2ogr`. It can be used from Python with the `pyogrio` or `fiona` libraries or from R with the `sf` library.
+
+OGR is installed as part of [GDAL](#gdal).
+
+#### Overviews
+
+Downsampled (aggregated) data intended for visualization and stored as part of a file format. Overviews are part of the [COG](#cloud-optimized-geotiff-cog) specification, and allow reading "zoomed out" data without needing to read and downsample from "full resolution" data.
+
+#### Parquet
+
+A file format for tabular data with [internal chunking](#chunk) and [internal compression](#internal-compression).
+
+#### PDAL
+
+The _Point Data Abstraction Library_, a widely-used open-source library for converting between different [point cloud data](#point-cloud-data) formats and managing point cloud data.
+
+#### PMTiles
+
+A type of [cloud-optimized](#cloud-optimized) [archive format](#archive-format) for tiled data. It can be used with either [vector](#vector-data) or [raster](#raster-data) tiled data, but is most often used with [Mapbox Vector Tile](#mapbox-vector-tile) files. The individual tiles in a PMTiles file are accessible via [HTTP range requests](#http-range-request).
+
+#### Point Cloud Data
+
+A type of geospatial data storing three dimensional point locations along with attributes for each point. Point cloud data may come from LIDAR sensors or other photogrammetry, and may represent three-dimensional terrain or buildings.
+
+#### PROJJSON
+
+A [projection definition](#coordinate-reference-system-crs) that uses JSON for encoding.
+
+The specification is [defined as part of the PROJ project](https://proj.org/en/9.3/specifications/projjson.html) and used as part of the [GeoParquet](#geoparquet) [vector file](#vector-data) format.
+
+#### Random access
+
+The ability to quickly fetch _part_ of a file without reading the entire file.
+
+For example, consider videos on YouTube. If you select to watch a video starting from the ten minute mark, YouTube does not need to download the video up to that point. Rather it is able to use the [metadata](#metadata) from the video file to know what byte range in the video file corresponds to the ten minute mark, and then use [HTTP range requests](#http-range-request) to download only the part of the video you've reqested.
+
+The ability to perform efficient random access over a network is a core part of what makes data [cloud-optimized](#cloud-optimized).
+
+#### Raster data
+
+A type of geospatial data that stores regularly-gridded data. This often comes from aerial or satellite imagery sensors.
```

Comment:
how/do we differentiate from a regular grid of points?

Reply:
I'm not sure what you mean. Do you just mean to clarify that `data` can have a significance of either a point at the center of the raster cell or the area contained by the raster cell?

Reply:
More so that Rasters have Cells of known size.